src/content/guides/multi-nginx/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Running Multiple NGINX Ingress Controllers
+description: Deploy multiple NGINX ingress controllers in a Kubernetes cluster to separate different ingress traffic classes.
+date: 2020-08-05
+type: page
+tags: ["tutorial"]
+---
+
+# Multi-NGINX ingress controller support
+
+NGINX ingress controller handles [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) resources, routing the traffic from outside of the Kubernetes cluster to services within the cluster. Starting with App version 1.8.0 one can install multiple NGINX ingress controllers in same Kubernetes cluster. Here we cover common cases where having multiple NGINX ingress controllers can be useful.
+
+Most of the NGINX configuration options have NGINX wide defaults but can also be overriden on per Ingress resource level. In each of the covered multi-NGINX use cases, one essentially installs second NGINX with different NGINX global-only configuration, i.e. configuration that applies to all Ingress resources managed by that NGINX installation which cannot be customized on per-Ingress resource level.
+
+## Separating public from internal ingress traffic
+
+Having some components in multi-tierd architecture or back-office applications not accessible via public internet is a common requirement for security or performance reasons. This is how one can achieve it by using multi-NGINX ingress controller support:
+
+- Deploy one NGINX IC App using default settings, for making selected cluster services accessible via the public internet
```

Comment:
Would it be helpful to link to docs here on default settings?

Reply:
I've added link to our app/chart configuration options.